migrations/versions/0429_go_live_templates.py

```diff
@@ -0,0 +1,158 @@
+"""
+
+Revision ID: 0429_go_live_templates
+Revises: 0428_letter_rates_errata
+Create Date: 2023-10-31 13:23:51.260909
+
+"""
+import textwrap
+
+from alembic import op
+from flask import current_app
+
+revision = "0429_go_live_templates"
+down_revision = "0428_letter_rates_errata"
+
+continue_template_id = "62f12a62-742b-4458-9336-741521b131c7"
+continue_template_content = textwrap.dedent(
+    """
+    ((body))
+    """
+)
+
+reject_template_id = "507d0796-9e23-4ad7-b83b-5efbd9496866"
+reject_template_content = textwrap.dedent(
+    """
+Hi ((name))
+
+# Your request to go live was rejected
+
+You sent a request to go live for a GOV.UK Notify service called ‘((service_name))’.
+
+((organisation_team_member_name)) at ((organisation_name)) rejected the request for the following reason:
+
+((reason))
```

Comment:
@samuelhwilliams Can we only add the inset text (`^`) formatting once this exists in the Notify admin interface?

Reply:
`reason` comes through to us here with the inset text formatting baked in by admin.

However I might tweak that so that it happens here instead, so will make a small change...